src/modules/project/pages/projectView/hooks/useProjectGoalForm.tsx

```diff
@@ -142,6 +151,13 @@ export const useProjectGoalForm = (
               emojiUnifiedCode: formData.emojiUnifiedCode,
             },
           },
+          onError(error) {
+            toast({
+              title: 'Error creating project goal',
```

Comment:
Please add these to translation files if not added already.

Note: All things inside toast are already wrapped inside t(''), so you don't need to wrap them before sending though. So it's already good, 

You only need to add these texts to english.json

Reply:
Hey @sajald77 I solved this!